core/src/main/java/org/apache/iceberg/SnapshotProducer.java

```diff
@@ -282,6 +283,29 @@ public Snapshot apply() {
       throw new RuntimeIOException(e, "Failed to write manifest list file");
     }
 
+    Long addedRows = null;
+    Long lastRowId = null;
+    if (base.rowLineage()) {
```

Comment:
maybe we should call this `rowLineageEnabled()`? It would certainly read better

Reply:
Sounds good to me